package.json

```diff
@@ -1,97 +1,120 @@
 {
-  "name": "cvs-svc-document-retrieval",
-  "version": "1.0.0",
-  "description": "Microservice that retrieves documents",
-  "main": "handler.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/dvsa/cvs-document-retrieval.git"
-  },
+  "name": "cvs-svc-template",
+  "version": "1.1.1",
```

Comment:
did we make a release? I couldn't see any details in the changelog file
I think we can revert it back to 1.0.0 and if it is the case that we need to make a release so we could run the `release` script to publish changes, notes, etc...